src/Microsoft.IdentityModel.Validators/AadTokenValidationParametersExtension.cs

```diff
@@ -109,6 +138,45 @@ internal static bool ValidateIssuerSigningKey(SecurityKey securityKey, SecurityT
             return true;
         }
 
+        /// <summary>
+        /// Stores the cloud instance name of the signing key in a property bag and validates it.
+        /// </summary>
+        /// <param name="securityKey">The <see cref="SecurityKey"/> that signed the <see cref="SecurityToken"/>.</param>
+        /// <param name="configuration">The <see cref="BaseConfiguration"/> provided.</param>
+        /// <param name="cloudInstanceName">The cloud instance name to validate against.</param>
+        /// <returns><c>true</c> if the cloud instance name of the signing key is valid; otherwise, <c>false</c>.</returns>
+        internal static bool StoreValidateSigningKeyCloudInstanceName(SecurityKey securityKey, BaseConfiguration configuration, string cloudInstanceName)
```

Comment:
nit: Consider dropping the "Store" prefix as putting the instance in a property bag is an implementation detail.

Reply:
fixed